sections/Deals/Vesting/VestingDealSection/VestingSection.tsx

```diff
@@ -37,12 +42,11 @@ const VestingDeal: FC<VestingDealProps> = ({
 }) => {
 	const { walletAddress, signer } = Connector.useContainer();
 	const { monitorTransaction } = TransactionNotifier.useContainer();
-	const { txState, setTxState, gasPrice, setGasPrice, setTxType, txType } =
-		TransactionData.useContainer();
-	const [isMaxValue, setIsMaxValue] = useState<boolean>(false);
-	const [inputValue, setInputValue] = useState(0);
+	const { setTxState, gasPrice, setTxType } = TransactionData.useContainer();
 	const [gasLimitEstimate, setGasLimitEstimate] = useState<GasLimitEstimate>(null);
 
+	useEffect(() => setTxType(TransactionType.Vest), [setTxType]);
```

Comment:
We really need to do something about this.

Reply:
Done